.github/workflows/ci.yml

```diff
@@ -11,12 +11,18 @@ jobs:
       - name: Flake8
         run: flake8 . --exclude=*/__init__.py,osp/core/ontology/docs/EMMO/* --ignore=W503 --docstring-convention=google
 
+      - name: isort
+        run: isort --profile black --filter-files --line-length 79 --check-only .
+
+      - black:
+        run: black --check --verbose --diff --line-length 79 .
+
```

Comment:
Here I would just run `pre-commit run --all-files`.
That way there is no need to maintain the different tasks

Reply:
Isn't the client already doing that with the hook? I just put this to ensure that a _rogue_ developer that skips the hook fails to pass the CI.

Reply:
Yes, but if you decide to add a new hook in a later stage, you would have to update this. Instead, you can make GitHub actions run based on the config file, and the pipeline will fail if something is wrong